11ty/definitions/-misia.md

```diff
@@ -0,0 +1,44 @@
+---
+title: -misia
+slug: -misia
+speech: noun
+defined: true
+sub_terms:
+  - text: Fat
+    full_title: fatmisia
+  - text: Islam
+    full_title: fatmisia
+  - text: Trans
+  full_title: transmisia
+reading:
+  - text: Beatley Library Anti-Oppression Guide
+    href: https://simmons.libguides.com/anti-oppression#s-lib-ctab-10174165-1
+  - text: 'Anti-Oppression: Anti-Fatmisia'
+    href: https://simmons.libguides.com/anti-oppression/anti-fatmisia
+---
+---
+from Greek for hate or hatred
+
+## Use
+
+-Misia can be appended to minoritised identifiers that are targeted for hate, such as to fat (fatmisia), trans (transmisia), or Islam (Islamomisia).
```

Comment:
Maybe remove the “to” in this bit (would read a bit smoother to me): “such as to fat”.